### PR TITLE
Add a rake task to invite mentors with unsent invites

### PR DESCRIPTION
**Pull request by kicferk1**
## Ticket and context

I want to send invite emails to all mentors who are on our system. The syncing script made an email for each of them. And this will send them. If they are doing FIP, the `send!` method shouldn't work. 

### Is there anything weird that code reviewer should know?
Nope, everything is straightforward.

### CODE REVIEW

File: lib/tasks/invite_mentors.rake [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+namespace :invites do
+  desc "Invite mentor users"
+  task send_mentor_invites: :environment do
+    puts("Sending mentor invites...")
+
+    unsent_emails = InviteEmailMentor.where(sent_at: nil)
+    unsent_emails.each do
+      email.send!(force_send: true)
+    rescue Notifications::Client::RateLimitError
+      sleep(1)
```
Comment (xswoodsx) [inline lib/tasks/invite_mentors.rake:12]: sleeps the equivalent of setInterval in JS? Why's it needed here? Is it incase we're sending them to quickly after each other and this just adds a delay to allow the task to continue without failing completely? If that's right, what's the rate limit?

Comment (xswoodsx) [reply lib/tasks/invite_mentors.rake:12]: 360 per second by the looks of it

**Review by xswoodsx — APPROVED** [review]

**Comment by kicferk1** [comment]
Apparently we don't need this just yet - we want to send the emails to a limited subset of our mentors due to DQT validation concerns (it is coming down the line and we don't want to have to do it for too many people).

File: lib/tasks/invite_mentors.rake (hunk 1 above)
Comment (kicferk1) [reply lib/tasks/invite_mentors.rake:12]: Yeah, that is what it does. I think 1 second is enough, I stole the idea from R&P who have a lot more experience sending vast amounts of emails.

**Pull request closed without merging**